librz/util/print_code.c

```diff
@@ -95,6 +95,9 @@ static void print_c_code(RzPrint *p, ut64 addr, const ut8 *buf, int len, int ws,
 }
 
 RZ_API void rz_print_code(RzPrint *p, ut64 addr, const ut8 *buf, int len, char lang) {
+	if (!p || !buf) {
```

Comment:
Use `rz_return_val_if_fail` here. If you are passing a NULL RzPrint you are doing something wrong, so better be warned than just seeing nothing happening maybe later.